en/docs/fragments/configure-resident-idp.md

```diff
@@ -0,0 +1,17 @@
+## Configure the resident IdP
+
+1. Log in to the [Management Console](https://localhost:9443/carbon/) using admin/admin credentials. 
```

Comment:
If there is no server up and running on this host and port, then this link will not work. Hence maybe let's link this to the Management console page in the admin guide.